src/network/network_admin.cpp

```diff
@@ -137,7 +137,6 @@ NetworkRecvStatus ServerNetworkAdminSocketHandler::SendError(NetworkErrorCode er
 
 	p->Send_uint8(error);
 	this->SendPacket(p);
-
```

Comment:
Please in PRs always try to minimize the change. Removing these whiteline (as for the 2 others in this PR) carries no weight, and only distracts from your change. So it is better to remove them.

Otherwise we have constant changes in newlines in PRs, depending on the mood of the author ;) :D